crates/ide-assists/src/handlers/unqualify_method_call.rs

```diff
@@ -76,10 +79,49 @@ pub(crate) fn unqualify_method_call(acc: &mut Assists, ctx: &AssistContext<'_>)
                 edit.insert(close, ")");
             }
             edit.replace(replace_comma, format!(".{method_name}("));
+            add_import(&path, ctx, edit);
         },
     )
 }
 
+fn add_import(
+    path: &ast::Path,
+    ctx: &AssistContext<'_>,
+    edit: &mut ide_db::source_change::SourceChangeBuilder,
+) {
+    let qualifier = path.qualifier().unwrap();
```

Comment:
Let's try to avoid these `unwrap`s. I'm not sure if they can trigger, but it's probably fine to `return` in these cases.

Reply:
Actually it will not failed at this `unwrap`, casuse line 44 have (`_ = path.qualifier()?;`) verified. I will refactor it later.